docs/book/src/developers/development.md

```diff
@@ -151,6 +153,11 @@ development will span both CAPZ and CAPI, then follow the [CAPI and CAPZ instruc
 
 #### Tilt for dev in CAPZ
 
+<aside class="note warning">
+<h1> Warning </h1>
+If you are looking to leverage the internal LB for internode communication in your workload cluster, please follow the instructions in the [Tilt for dev using internal ILB for internode communication](#tilt-for-dev-using-internal-ilb-for-internode-communication) section.
```

Comment:
Let's rephrase "internode communication" to "local node-apiserver communication"

Reply:
the locality is the cluster

how about "cluster VNET node-apiserver communication"

Reply:
"cluster VNET node-apiserver communication" sounds good to me!

Reply:
Added via 1b810e62730e82b2768ecf4ef457ddd42465c3b2